src/components/ItemPopover.vue

```diff
@@ -8,6 +8,13 @@
   >
     <div class="popup d-flex flex-column align-items-center">
       <h6 class="title">{{item.name}}</h6>
+      <p v-if="item.equipmentSlot" class="equipment-slot-name text-uppercase">{{item.equipmentSlot}}</p>
+      <equipment-slot
+        v-if="item.equipmentSlot"
+        class="equipment-slot-icon"
+        :equipmentSlot="item.equipmentSlot"
+        :icon="require(`@/assets/art/combat/equipment/${item.equipmentSlot}.png`)"
+      />
       <span v-if="item.healAmount" class="mt-1">Heals +{{item.healAmount}} HP</span>
       <span v-if="item.equipmentSlot" class="mt-1">Equippable</span>

```

Comment:
I'd be fine if you just got rid of this, the slot itself is more specific anyway.

Reply:
✅

Good input :)